### PR TITLE
zmc: hosting your website on github page video demos

### PR DESCRIPTION
**Pull request by Sbonelo01**
Related issues: https://www.notion.so/hosting-your-page-on-github-pages-Show-learners-how-to-set-up-their-github-page-9790ebda1df64ccb98abffd8cb64b429?pvs=4

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Comment by Ngoakor12** [comment]
@Sbonelo01 PC demo looks good to me.

Please check my Android demo

File: content/zmc-challenges/first-website/hosting-your-page-on-github-pages/_index.md [hunk 1]
```diff
@@ -56,61 +58,67 @@ From now on you'll be able to interact with your repo by navigating to it in you
...
-Navigate to your website. Does it look the way you expected? 
+Navigate to your website. Does it look the way you expected?
 
 **Note:** Sometimes it can take a few minutes for the website to go live. If it's not working then wait a few minutes and check again.
 
-### 7. Submit your work 
+We have also created some video demos to walk you through some of the steps above:
+
+- [PC/Computer demo](https://www.youtube.com/watch?v=auWR5JOiZt4)
+- [Android phone demo](https://drive.google.com/file/d/1yBKPaq0yjMIqZxEgPSTgNdlkGTsswjqh/view?usp=sharing)
```
Comment (Sbonelo01) [inline content/zmc-challenges/first-website/hosting-your-page-on-github-pages/_index.md]: Good job. 
I am not sure about the location you chose to store this video on. Every other video demo, including those made by sheena, have been recorded with Streamyard and made available on the Umuzi YouTube channel

**Review by Sbonelo01 — COMMENTED** [review]
Good job. Not sure about where the video is stored.

**Review by Sbonelo01 — COMMENTED** [review]
Nicely done! Thanks!

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**